src/lazyfree.c

```diff
@@ -72,8 +72,10 @@ void lazyFreeLuaScripts(void *args[]) {
 /* Release the functions ctx. */
 void lazyFreeFunctionsCtx(void *args[]) {
     functionsLibCtx *functions_lib_ctx = args[0];
+    dict *engs = args[1];
     size_t len = functionsLibCtxFunctionsLen(functions_lib_ctx);
```

Comment:
So we will still do the lua_uref from the background thread, but you consider it safe because you know that this lua vm will not be touched anymore from the main thread, is that right?